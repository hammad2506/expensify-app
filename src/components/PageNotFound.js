import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => (
    <div>
        <h3>404 Page NOT FOUND</h3>
        <Link to='/'> Home Page </Link>
    </div>
);

export default PageNotFound;