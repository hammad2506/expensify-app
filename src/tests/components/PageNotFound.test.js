import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../components/PageNotFound';

test("Testing PAGE NOT FOUND Component", () => {
    const wrapper =  shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});