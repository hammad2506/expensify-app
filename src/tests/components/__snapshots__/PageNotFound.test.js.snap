// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PAGE NOT FOUND Component 1`] = `
<div>
  <h3>
    404 Page NOT FOUND
  </h3>
  <Link
    replace={false}
    to="/"
  >
     Home Page 
  </Link>
</div>
`;
